packages/block-library/src/navigation/index.php

```diff
@@ -1355,10 +1355,10 @@ function block_core_navigation_get_most_recently_published_navigation() {
  * of those hooked blocks should be exempted from insertion.
  *
  * @param array   $inner_blocks Parsed inner blocks of a Navigation block.
- * @param WP_Post $post         `wp_navigation` post object corresponding to the block.
+ * @param WP_Post $post        `wp_navigation` post object corresponding to the block.
```

Comment:
The arguments are not aligned now no?

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/58378/commits/14e856c621cf57df45b46b6b15e842daf1479d94

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/58378/commits/14e856c621cf57df45b46b6b15e842daf1479d94